### DISCUSSION TITLE
Cumulative VP fix

### DISCUSSION THREAD

**Discussion by N6REJ** [hythloday/VenturePlanSoDMissions#67] [Q&A]
I vaguely recall someone had done extra work and fixed the UI quite a bit.  Where did that get put?

**Comment by TruthNZ** [comment]
I think most updates are now over here:
https://github.com/Divergentcurl/VenturePlanSoDMissions/tree/cumulative-fixes

[Edit] It's worth noting that the "Getting it running" part has a little more you need to do now.

**Reply by k1ck3r** [reply]
yep. there are
